### ISSUE TITLE
send2trash unsupported in Debian 9

### ISSUE THREAD

**Issue by Head-on-a-Stick** [nwg-piotr/azote#12]
Debian 9 (stretch) doesn't have a python3-send2trash package so the package won't install in that version of Debian.

I could host a backported version in the `azote` repository but the hosting service I use won't allow the ~bpo package name suffix so the package won't be replaced by the Debian buster version if users upgrade their systems.

As such, I'd rather just drop Debian 9 support.

Is a *send2trash* option really needed in a wallpaper chooser?

Some simple, window manager based desktops don't have a Trash directory (mine doesn't) so the option is hardly universal.

Thoughts?

**Comment by nwg-piotr** [comment]
Surely the "Sent to trash" feature it's not a must, but personally I use it. The issue could be resolved, at least partially, this way:

1. remove the `python-send2trash` package dependency;
2. restore `install_requires=['send2trash']` in setup.py, to try installing with pip;
3. only draw the button if `import send2trash` command successful.

What do you think?

**Comment by Head-on-a-Stick** [comment]
Option number 3 looks best to me.

Option 1 would leave part of the program broken and option 2 doesn't make `pip` install stuff in my Debian systems—the `install_requires` components just add the appropriate Debian dependencies to the package (courtesy of the debhelper build system).

**Comment by nwg-piotr** [comment]
Well, it's a sequence of todos to resolve the issue, rather than a list of solutions to choose from.

**Comment by Head-on-a-Stick** [comment]
OK, sorry.

Adding send2trash to `install_requires` would make debhelper add python3-send2trash to the dependencies list when the package is built.

I think just dropping Debian 9 support is probably the sensible approach here, Debian 10 will be released in a couple of weeks anyway.

**Comment by nwg-piotr** [comment]
> Adding send2trash to install_requires would make debhelper add python3-send2trash to the dependencies list when the package is built.

I'm not sure: will this resolve the issue on Debian9 or not?

**Comment by Head-on-a-Stick** [comment]
No, the only way to make the current version work with Debian 9 is to backport the python3-send2trash program and I don't want to do that with the package hosting service I'm using because it might interfere with upgrades from Debian 9 to Debian 10.

Just tried adding send2trash to `install_requires` and the package building system tells me this:
```
I: dh_python3 pydist:220: Cannot find package that provides send2trash. Please add package that provides it to Build-Depends or add "send2trash python3-send2trash" line to debian/py3dist-overrides or add proper  dependency to Depends by hand and ignore this info.
```

**Comment by nwg-piotr** [comment]
Resolved in v0.1.2: if the python send2trash module not available (import fails), the button won't be drawn. Please **remove the python-send2trash dependency where the package unavailable**.
